xivo_dao/alchemy/queue.py

```diff
@@ -88,6 +89,22 @@ class Queue(Base, AsteriskOptionsMixin):
     announce_position_limit = Column('announce-position-limit', Integer, nullable=False, server_default='5')
     defaultrule = Column(String(1024))
 
+    groupfeatures = relationship(
+        'GroupFeatures',
+        primaryjoin="""and_(Queue.category == 'group',
+                            Queue.name == GroupFeatures.name)""",
+        foreign_keys='Queue.name',
+        uselist=False,
+    )
+
+    queuefeatures = relationship(
+        'QueueFeatures',
+        primaryjoin="""and_(Queue.category == 'queue',
+                            Queue.name == QueueFeatures.name)""",
+        foreign_keys='Queue.name',
+        uselist=False,
+    )
+
```

Comment:
You should add a joinloaded to the asterisk_conf_dao query to avoid slow down query with many queues

Reply:
fixed, thanks